### PR TITLE
defining workload of a course instance

### PR DESCRIPTION
**Pull request by philbarker**
defining workload of a course instance for #1909

### CODE REVIEW

File: data/ext/pending/issue-1909.rdfa [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div>
+  <!-- course workload, issue 1909 -->
+  <div typeof="rdf:Property" resource="http://schema.org/courseWorkload">
+      <span class="h" property="rdfs:label">courseWorkload</span>
+      <span property="rdfs:comment">The amount of work expected of students on the course, often provide as a figure per week or per month, and may be broken down by type. For example, "2 hours of lectures, 1 hour of lab work and 3 hours of independent study per week".</span>
```
Comment (thadguidry) [inline data/ext/pending/issue-1909.rdfa]: "often provide**d**"

Comment (philbarker) [reply data/ext/pending/issue-1909.rdfa]: Thanks, fixed that along with a change suggested after comment by @stuartasutton

**Comment by philbarker** [comment]
@RichardWallis just pinging you in case you missed that I had made the change you request. Perdona la molestia.

**Comment by danbri** [comment]
Merged. Can we have a nice sentence for docs/releases.html too please? Is "Adding a courseWorkload property for use with Course type." ok?

p.s. is there a case for allowing QuantitativeValue too?

**Comment by danbri** [comment]
We have a problem here, the new courseWorkload property is not showing up in the drafting site (webschemas) for some reason. I tried adding more markup into the definition to match the (verbose) way other pending terms are associated with Github issues.

* RDFa: https://github.com/schemaorg/schemaorg/blob/master/data/ext/pending/issue-1909.rdfa 

Parsing this on commandline looks legit,

```
python2.7 ./scripts/rdfa2nt data/ext/pending/issue-1909.rdfa
<http://schema.org/courseWorkload> <http://schema.org/rangeIncludes> <http://schema.org/Text> .
<http://schema.org/courseWorkload> <http://www.w3.org/2000/01/rdf-schema#comment> "The amount of work expected of students taking the course, often provided as a figure per week or per month, and may be broken down by type. For example, \"2 hours of lectures, 1 hour of lab work and 3 hours of independent study per week\"." .
<http://schema.org/courseWorkload> <http://purl.org/dc/terms/source> <https://github.com/schemaorg/schemaorg/issues/1909> .
<http://schema.org/courseWorkload> <http://schema.org/domainIncludes> <http://schema.org/CourseInstance> .
<http://schema.org/courseWorkload> <http://www.w3.org/2000/01/rdf-schema#label> "courseWorkload" .
<http://schema.org/courseWorkload> <http://schema.org/isPartOf> <http://pending.schema.org> .
<http://schema.org/courseWorkload> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/1999/02/22-rdf-syntax-ns#Property> .
<http://schema.org/courseWorkload> <http://schema.org/category> "issue-1909" .
```

**Comment by RichardWallis** [comment]
Fixed by ```running buildTermConfig.sh``` to introduce issue-1909.rdfa into the config.

Should appear after next deploy

**Pull request merged**